crawlers/ten_ago_crawler.py

```
@@ -0,0 +1,138 @@
+from selenium import webdriver
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)